corehq/apps/app_manager/views/releases.py

```diff
@@ -352,3 +371,28 @@ def page_context(self):
     @property
     def page_url(self):
         return reverse(self.urlname, args=[self.domain, self.first_app_id, self.second_app_id])
+
+class LanguageProfilesView(View):
+    urlname = 'build_profiles'
+
+    @method_decorator(require_can_edit_apps)
+    @method_decorator(requires_privilege(privileges.BUILD_PROFILES))
+    def dispatch(self, request, *args, **kwargs):
+        return super(LanguageProfilesView, self).dispatch(request, *args, **kwargs)
+
+    def post(self, request, domain, app_id, *args, **kwargs):
+        profiles = json.loads(request.body).get('profiles')
+        app = Application.get(app_id)
+        build_profiles = {}
+        if profiles:
+            for profile in profiles:
+                id = profile.get('id')
+                if not id:
+                    id = uuid.uuid4().hex
+                build_profiles[id] = BuildProfile(langs=profile['langs'], name=profile['name'])
```

Comment:
It looks like this replaces all existing build profiles with a new set of profiles. I assume it's possible that some may have been deleted. Does it matter if other resources such as attachments are orphaned as a result of this assignment? Should they be deleted here?